lib/fs.js

```diff
@@ -145,8 +145,8 @@ function makeCallback(cb) {
     throw new errors.TypeError('ERR_INVALID_CALLBACK');
   }
 
-  return function() {
-    return cb.apply(undefined, arguments);
+  return function(...arguments) {
+    return cb.apply(undefined, args);
```

Comment:
On that note, could we change this to `Reflect.apply(cb, undefined, args);`? Thank you! That will mean we no longer depend on the user provided `apply`.

Reply:
@apapirovski : Please take a look now. Thanks !